components/infobox/commons/infobox_person.lua

```diff
@@ -138,6 +139,7 @@ function Person:createInfobox()
 		Title{name = (args.informationType or 'Player') .. ' Information'},
 		Cell{name = 'Name', content = {args.name}},
 		Cell{name = 'Romanized Name', content = {args.romanized_name}},
+		Cell{name = 'Abbreviations', content = {args.abbreviations}},
```

Comment:
imo better suited for custom
don't see much usage on esports wikis

Reply:
make the name cells customizable and add it in there

Reply:
in commons:
```lua
		Customizable{id = 'name', children = {
			Cell{name = 'Name', content = {args.name}},
			Cell{name = 'Romanized Name', content = {args.romanized_name}},
			}
		},
```

and in the custom in the injector parse you can then just `if id == 'name' then table.insert(widgets, ...) end`